src/pages/Trade/components/ChartCard/PairListPanel/MarketsTable.tsx

```diff
@@ -103,6 +113,23 @@ const StyledTable = styled(Table)`
 	}
 `;
 
+const SnowflakeIconWrapper = styled.div`
```

Comment:
both of these seem duplicate, could you move the first `SnowflakeIconWrapper` and `StyledSnowflakeIcon` to a common file, and then style these ones to match?